test/development/app-dir/error-overlay/async-client-component/app/client/page.tsx

```diff
@@ -0,0 +1,5 @@
+'use client'
+
+export default async function Page() {
```

Comment:
Is it available once you destructure props in this Component or use a Hook e.g. `React.useState()`?

Reply:
Nope, neither works